src/redux/reducers/app.ts

```diff
@@ -12,5 +14,9 @@ const initialState: AppState = {
 };
 
 export function appReducer(state: AppState = initialState, action: Action): AppState {
-  return state;
+  if (drawingCanvasActions.indexOf(action.type) >= 0) {
```

Comment:
My recommendation is to not filter the actions in this way, but to always return the passed in state if the reducer doesn't handle the action. Also, since the drawing canvas reducer looks to only be modifying the shapes array, you could do something like this:

```ts
appReducer(..) {
  return {
    shapes: drawingCanvasReducer(state.shapes, action);
  }
}
```